README.md

```diff
@@ -192,16 +194,16 @@ The mapping can be described mathematically with the following formulas:
 
 By joint 6 the sign of the relative positions must be considered, and to assure continuous joint values without leaps, two cases should be considered:
 
-If $\lvert$&Delta;x $\rvert$ $\lt$ $\lvert$&Delta;y $\lvert$:
+![](https://latex.codecogs.com/svg.latex?&space;\lvert\Delta{}x\rvert<\lvert\Delta{}y\lvert:)
 
 ![](https://latex.codecogs.com/svg.latex?&space;\Theta{}_6=\mathrm{sgn}(y)*\mathrm{atan2}\(\Delta{}z_{wrist},\sqrt{\Delta{}x_{wrist}^2+\Delta{}y_{wrist}^2}\))
 
-If $\lvert$&Delta;x $\rvert$ $\ge$  $\lvert$&Delta;y $\lvert$:
+![](https://latex.codecogs.com/svg.latex?&space;\lvert\Delta{}x\rvert\ge\lvert\Delta{}y\lvert:)
 
 ![](https://latex.codecogs.com/svg.latex?&space;\Theta{}_6=\mathrm{sgn}(x)*\mathrm{atan2}\(\Delta{}z_{wrist},\sqrt{\Delta{}x_{wrist}^2+\Delta{}y_{wrist}^2}\))
 
 
-The trigonometric calculations and coordinate transformations could be easily implemented with math and eigen packages, the resulting joint angles sent to the joint controller node for execution. The implementation is done in the map_arm package, which filters the incoming markers for the necessary ones and calculates the inverse kinamtics, finally publishing the reference joint states.
+The trigonometric calculations and coordinate transformations could be easily implemented with math and eigen packages, the resulting joint angles sent to the joint controller node for execution. The implementation is done in the map_arm package, which filters the incoming markers for the necessary ones and calculates the inverse kinematics, finally publishing the reference joint states.
 
 
 ## Modularity
```

Comment:
I would change on of blue or purple to another color, because they would be easier to be differentiated that way.